l10n_be_cooperator_national_number/models/res_partner.py

```diff
@@ -32,7 +35,27 @@ def get_be_national_register_number(self):
             return id_number.name
         return None
 
-    def update_belgian_national_number(self, national_number):
+    @api.model
+    def check_be_national_register_number(self, national_number, error=False):
+        """Verify whether the national number is valid. Returns True if invalid,
+        which is consistent with validate_l10n_be_national_registry_number.
```

Comment:
i really think that returning true if something is invalid is bad practice (although `partner_identification` does it that way). please rename the function to `validate_*` and return true or false if valid or not, or raise the error if invalid and error is true.

Reply:
OK, agreed, this tripped me up earlier as well.